src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * {@code DeleteCommand} is a child of {@code Command} object.
+ *      On execution, it will delete a given index from the task list.
+ */
+public class DeleteCommand extends Command {
+
+    private int idx;
```

Comment:
perhaps you want to use the name index instead of idx to improve comprehensiveness